components/Typography/examples/TypographyMaterialScalableStylesViewController.m

```diff
@@ -0,0 +1,173 @@
+  NSLog(@"New size category : %@", sizeCategory);
+
+  // Update font array to reflect new size category
+  _styleFonts = @[
+    [_typography.body1 mdc_scaledFontAtDefaultSize],
+    [_typography.body1 mdc_scaledFontForCurrentSizeCategory],
+    [_typography.body2 mdc_scaledFontAtDefaultSize],
+    [_typography.body2 mdc_scaledFontForCurrentSizeCategory],
+    [_typography.caption mdc_scaledFontAtDefaultSize],
+    [_typography.caption mdc_scaledFontForCurrentSizeCategory],
+    [_typography.button mdc_scaledFontAtDefaultSize],
+    [_typography.button mdc_scaledFontForCurrentSizeCategory],
+    [_typography.overline mdc_scaledFontAtDefaultSize],
+    [_typography.overline mdc_scaledFontForCurrentSizeCategory],
+    [_typography.subtitle1 mdc_scaledFontAtDefaultSize],
+    [_typography.subtitle1 mdc_scaledFontForCurrentSizeCategory],
+    [_typography.subtitle2 mdc_scaledFontAtDefaultSize],
+    [_typography.subtitle2 mdc_scaledFontForCurrentSizeCategory],
+    [_typography.headline1 mdc_scaledFontAtDefaultSize],
+    [_typography.headline1 mdc_scaledFontForCurrentSizeCategory],
+    [_typography.headline2 mdc_scaledFontAtDefaultSize],
+    [_typography.headline2 mdc_scaledFontForCurrentSizeCategory],
+    [_typography.headline3 mdc_scaledFontAtDefaultSize],
+    [_typography.headline3 mdc_scaledFontForCurrentSizeCategory],
+    [_typography.headline4 mdc_scaledFontAtDefaultSize],
+    [_typography.headline4 mdc_scaledFontForCurrentSizeCategory],
+    [_typography.headline5 mdc_scaledFontAtDefaultSize],
+    [_typography.headline5 mdc_scaledFontForCurrentSizeCategory],
+    [_typography.headline6 mdc_scaledFontAtDefaultSize],
+    [_typography.headline6 mdc_scaledFontForCurrentSizeCategory],
+  ];
+
+  [self.tableView reloadData];
+}
+
+#pragma mark - UITableViewDataSource
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
+  return _strings.count;
+}
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+  return _styleNames.count;
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView
+         cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+  UITableViewCell *cell = [self.tableView dequeueReusableCellWithIdentifier:@"cell"];
+  if (cell == nil) {
+    cell = [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleSubtitle
+                                  reuseIdentifier:@"cell"];
+  }
+  cell.textLabel.text = _strings[indexPath.section];
+  cell.textLabel.font = _styleFonts[indexPath.row];
+  cell.textLabel.numberOfLines = 0;
+  cell.textLabel.lineBreakMode = NSLineBreakByWordWrapping;
+
+  if (cell.textLabel.font.pointSize > 100 && indexPath.section == 0) {
+    cell.textLabel.text = @"MDC";
+  }
+
+  NSString *detail = [NSString
+      stringWithFormat:@"%@ @ %.0f pt", _styleNames[indexPath.row], cell.textLabel.font.pointSize];
+  cell.detailTextLabel.text = detail;
+  cell.detailTextLabel.font = [_typography.caption mdc_scaledFontAtDefaultSize];
```

Comment:
Why wouldn't this scale with Dynamic Type? A user having trouble reading the other text probably has trouble reading `caption` text.

Reply:
Intentionally turned off DT in this case to make is easier to compare and contrast various size categories.
If it grows, that is fine, but I *don't* want it to shrink, so it's disabled.